client/my-sites/checkout/cart/cart-item.jsx

```diff
@@ -122,7 +121,7 @@ export class CartItem extends React.Component {
 		if ( cartItem && cartItem.product_cost ) {
 			return (
 				<span>
-					{ ! ( isEnabled( 'upgrades/2-year-plans' ) && this.isDomainProductDiscountedTo0() ) ? (
+					{ ! this.isDomainProductDiscountedTo0() ? (
```

Comment:
Small nitpick, but I think spelling out `Zero` in this case would have made the method name more readable.